script.js

```diff
@@ -1,4 +1,146 @@
+var attempts = 0;
+var wins = 0;
+var playerName = "";
+var gamemode = "normal";
+var currentWinner = "";
```

Comment:
Great setting up of your global values Han Fan

In future you may want to explore something know as SCREAM_CASE constants

I can touch more on this as the days go by, but it's simply another way to name variables that remain unchanged through out the program eg.
`
var NORMAL_GAME_MODE = "normal"
var KOREAN_MODE = "korean"
`